Source/Core/EllipseGeometry.js

```diff
@@ -665,8 +665,8 @@ define([
         if (!defined(semiMinorAxis)) {
             throw new DeveloperError('semiMinorAxis is required.');
         }
-        if (semiMajorAxis <= 0.0 || semiMinorAxis <= 0.0) {
-            throw new DeveloperError('Semi-major and semi-minor axes must be greater than zero.');
+        if (semiMajorAxis < 0.0 || semiMinorAxis < 0.0) {
+            throw new DeveloperError('Semi-major and semi-minor axes must be not be negative.');
         }
         if (semiMajorAxis < semiMinorAxis) {
             throw new DeveloperError('semiMajorAxis must be greater than or equal to the semiMinorAxis.');
```

Comment:
@pjcozzi what do you think? I just suggested doing that because I thought we were trying to throw less developer errors based on incorrect data (since massive data sets tend to contain erroneous values here and there).